src/main/resources/db/migrations/V001__CreateSchemaTables.sql

```diff
@@ -0,0 +1,227 @@
+);
+
+CREATE TABLE public.roles (
+    id UUID PRIMARY KEY,
+    name VARCHAR(45) NOT NULL
+);
+
+CREATE TABLE public.permissions (
+    id UUID PRIMARY KEY,
+    name VARCHAR(45) NOT NULL
+);
+
+CREATE TABLE public.role_permission (
+    id UUID PRIMARY KEY,
+    role_id UUID REFERENCES roles(id),
+    permission_id UUID REFERENCES permissions(id)
+);
+
+
+CREATE TABLE public.app_access (
+    id UUID PRIMARY KEY,
+    user_id UUID REFERENCES users(id) NOT NULL,
+    court_id UUID REFERENCES courts(id) NOT NULL,
+    role_id UUID REFERENCES roles(id) NOT NULL,
+    last_access TIMESTAMPTZ,
+    active BOOLEAN DEFAULT TRUE,
+    deleted_at TIMESTAMPTZ DEFAULT NULL,
+    created_at TIMESTAMPTZ DEFAULT NOW(),
+    modified_at TIMESTAMPTZ DEFAULT NOW()
+);
+
+
+
+CREATE TABLE public.capture_sessions (
+    id UUID PRIMARY KEY,
+    booking_id UUID REFERENCES bookings(id) NOT NULL,
+    origin RECORDING_ORIGIN NOT NULL,
+    ingest_address VARCHAR(255),
+    live_output_url VARCHAR(100),
+    started_on TIMESTAMPTZ,
+    started_by_user_id UUID REFERENCES users(id),
+    finished_on TIMESTAMPTZ,
+    finished_by_user_id UUID REFERENCES users(id),
+    status RECORDING_STATUS,
+    deleted_at TIMESTAMPTZ DEFAULT NULL
+);
+
+CREATE TABLE public.recordings (
+    id UUID PRIMARY KEY,
+    capture_session_id UUID REFERENCES capture_sessions(id) NOT NULL,
+    version INT NOT NULL,
+    url VARCHAR(255) NOT NULL,
+    filename VARCHAR(255) NOT NULL,
+    created_at TIMESTAMPTZ NOT NULL,
+    duration TIME,
+    edit_instruction JSON,
+    deleted_at TIMESTAMPTZ
+);
+
+
+
+CREATE TABLE public.share_recordings (
+    id UUID PRIMARY KEY,
+    capture_session_id UUID REFERENCES capture_sessions(id) NOT NULL,
+    shared_with_user_id UUID REFERENCES users(id) NOT NULL,
+    shared_by_user_id UUID REFERENCES users(id) NOT NULL,
+    created_at TIMESTAMPTZ DEFAULT NOW(),
+    deleted_at TIMESTAMPTZ DEFAULT NULL
+);
+
+CREATE TABLE public.portal_access (
+    id UUID PRIMARY KEY,
+    user_id UUID REFERENCES users(id) NOT NULL,
+    password VARCHAR(45) NOT NULL,
+    last_access TIMESTAMPTZ,
+    status ACCESS_STATUS NOT NULL DEFAULT 'invitation_sent'::ACCESS_STATUS,
+    invitation_datetime TIMESTAMPTZ,
+    registered_datetime TIMESTAMPTZ,
+    deleted_at TIMESTAMPTZ DEFAULT NULL,
+    created_at TIMESTAMPTZ DEFAULT NOW(),
+    modified_at TIMESTAMPTZ DEFAULT NOW()
+);
+
+CREATE TABLE public.audits (
+    id UUID PRIMARY KEY,
+    table_name VARCHAR(25),
+    table_record_id UUID,
+    source AUDIT_LOG_SOURCE NOT NULL,
+    type AUDIT_LOG_TYPE NOT NULL,
+    category VARCHAR(100),
+    activity VARCHAR(100),
+    functional_area VARCHAR(100),
+    audit_details TEXT,
+    created_by VARCHAR(50),
+    created_at TIMESTAMPTZ NOT NULL DEFAULT NOW(), 
+    updated_at TIMESTAMPTZ NOT NULL DEFAULT NOW(),
+    deleted_at TIMESTAMPTZ DEFAULT NULL
+);
+
+ALTER TABLE new_schema.capture_sessions ADD FOREIGN KEY ("parent_recording_id") REFERENCES "recordings" ("id");
```

Comment:
Just realised you don't need the quotes. Also this should be `public` not `new_schema` :) 
```suggestion
ALTER TABLE public.capture_sessions ADD FOREIGN KEY (parent_recording_id) REFERENCES recordings (id);
```